src/ontology/vbo-edit.owl

```diff
@@ -48,6 +67,12 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#existsincount
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/vbo#existsincountryiso> "This annotation property refers to the country in which the breed exist, by using the UN ISO standard three-letter country codes. The numeric codes are available from https://www.iso.org/iso-3166-country-codes.html")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#existsincountryiso> "exists_in_country_ISOcode")
 
+# Annotation Property: <http://purl.obolibrary.org/obo/vbo#fully_recognized> (fully recognized)
+
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/vbo#fully_recognized> "Recognized breed or championship breed.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#fully_recognized> "fully recognized")
+SubAnnotationPropertyOf(<http://purl.obolibrary.org/obo/vbo#fully_recognized> <http://purl.obolibrary.org/obo/vbo#breed_recognition_status_type>)
```

Comment:
I suggest to model the recognition status as classes. You can basically leave the templates as they are, but you 

- make a new class "breed recognition status" (replacing breed_recognition_status_type)
- put all the `not_recognized` as proper VBO classes underneath
- leave `http://purl.obolibrary.org/obo/vbo#breed_recognition_status` as is

Reply:
done. I also created classes for other breed status (e.g. domestication status).